spec/controllers/spree/api/affirm_controller_spec.rb

```diff
@@ -21,7 +21,7 @@ module Spree
         expect(subject).to render_template("api/affirm/payload")
       end
 
-      it "renders the correct Affirm json payload" do
+      xit "renders the correct Affirm json payload" do
```

Comment:
We don't commit disabled specs unless the specs were previously merged into master, and need to be temporarily disabled as a regression.

Because we haven't committed to master a disabled spec should not exist.